src/shared/i18n/extension/empire/fr.ts

```diff
@@ -0,0 +1,311 @@
+            energyConsumption: 'Consommation d\'énergie',
+        },
+        denCapacity: {
+            header: 'Capacité de la cachette',
+        },
+        phalanxRange: {
+            header: 'Portée de la phalange',
+        },
+        discoveryMissions: {
+            header: 'Vitesse des missions de découverte',
+        },
+        playerClasses: {
+            header: 'Classes de joueur',
+
+            collector: {
+                productionBonus: 'Augmentation de la production des mines',
+                energyProductionBonus: 'Augmentation de la production d\'énergie',
+                transporterSpeedBonus: 'Bonus de vitesse pour les transporteurs',
+                transporterCargoBonus: 'Bonus de cargo pour les transporteurs',
+                crawlerProductionBonus: 'Bonus de production des foreuses',
+                geologistCrawlerAmountBonus: 'Bonus de foreuses utilisables avec le géologue',
+            },
+            discoverer: {
+                researchDuration: 'Durée des recherches',
+                increasedExpeditionFinds: 'Gain accru lors des expéditions',
+                largerPlanets: 'Planètes plus grandes lors de la colonisation',
+                additionalExpeditionSlots: 'Slot d\'expédition supplémentaires',
+                reducedExpeditionCombatChance: 'Réduction du risque de combats lors des expéditions',
+                phalanxRange: 'Augmentation de la portée de la phalange',
+            },
+            general: {
+                combatShipSpeed: 'Bonus de vitesse des vaisseaux de combat',
+                recyclerSpeed: 'Bonus de vitesse des recycleurs',
+                fuelConsumption: 'Consommation de carburant pour tous les vaisseaux',
+                increasedCargoRecyclersPathfinders: 'Augmentation de la capacité de charge des recycleurs/éclaireurs',
+                additionalCombatResearchLevels: 'Niveaux des recherches de combat supplémentaires',
+                additionalFleetSlots: 'Flottes supplémentaires',
+                additionalMoonFields: 'Champs de lune supplémentaires',
+            },
+
+            classBonus: 'Bonus de classe',
+            bonus: 'Bonus',
+            baseValue: 'Valeur de base',
+            valueWithBonuses: 'Valeur avec bonus',
+        },
+        planetBonuses: {
+            resourceProduction: {
+                header: 'Production de ressources',
+            },
+            lifeformResearchCostTime: {
+                header: 'Coût et temps de recherche des formes de vie',
+            },
+            wreckfield: {
+                header: 'Champ de débris',
+            },
+        },
+
+        topFinds: {
+            topFinds: 'Meilleures découvertes',
+            worstFinds: 'Pires découvertes',
+
+            size: 'Taille',
+            amount: 'Quantité',
+            date: 'Date',
+            artifacts: 'Artéfacts',
+            experience: 'Expérience',
+        },
+    },
+};
```

Comment:
when indentation fixed, add missing closing brace
```suggestion
    },
};
```